src/components/restaurant/Restaurant.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+
+
+class Restaurant extends Component {
+  render() {
+    return (
+      <div>
+        
+          <div className="container">

```

Comment:
how about splitting things into smaller components instead of creating a single page